pyrekordbox/config.py

```diff
@@ -274,11 +274,6 @@ def _get_rb_config(
         # `major_version` is compared to the version string
         rb_version = application_dirname.replace("rekordbox", "").strip()
         rb_version = rb_version.replace(".app", "")
-        if not rb_version.startswith(str(major_version)):
-            raise ValueError(
-                f"Major version is {major_version}, but the supplied application "
-                f"dirname is '{application_dirname}'"
-            )
         rb_prog_dir = pioneer_install_dir / application_dirname
```

Comment:
A little down from here (I can't comment on it :-/), in lines 278-282 there is the one catch when the custom .app dir would be non-existen. I think that is sufficient.

Further more down we have a lot of magic that tries to automatically find the latest version number, if there are several customly named rekordbox*.app directories lying around.

Suggestion: Let's remove all that: If rekordbox*.app was renamed, the user has to take care of it. No magic!

Reply:
Hey, thanks for taking the time looking into this!
I think you are right, we should just stick to checking if the directory exists. The version is only used in one place right now:

https://github.com/dylanljones/pyrekordbox/blob/d3d9a3a85216e73decae1b94bbf6b568ac78bf05/pyrekordbox/db6/database.py#L221-L225

An error is raised if no cached key is found and a Rekordbox version is used with the compiled .jsc files, so the key extraction does not work. But this might be unnecessary and there could be a better way of doing this. 

However, the magic *else branch* is required on Windows since there the default installation directory contains the version, for example `.../rekordbox 6.7.7`. I think the solution is to at least try to extract the version string, and if it fails just put a `unknown` into the version field of the config. This would require to remove the error linked above, but i think most people are using a newer version of Rekordbox anyway and thus have to require the key manually or download it via the CLI. as a side note, all of this is only necessary for old Rekordbox versions anyway, the installation directory of Rekordbox is only used for the key extraction until now. But i haven't looked into the installation files that much, there might be other useful stuff;) 

I will merge your PR and will make the changes mentioned above:) Thanks again for the help!